doc/user_guide/data.rst

```diff
@@ -627,6 +627,7 @@ data before usage in Altair using GeoPandas for example as such:
    customization
    configuration
    saving_charts
+   jupyter_chart
```

Comment:
I wonder if this section should be just under the section "Interactive Charts". It is definitely the closest to that section thematically even if it is not as central to Altair as some of the other section. I also wonder if we should call this section `JupyterChart Interactivity" or something similar to more clearly signal what it is about.

Reply:
Renamed to "JupyterChart Interactivity" in [5723c28](https://github.com/altair-viz/altair/pull/3137/commits/5723c287d05d39b679ee41833e7fe5232d4b839a)